apps/passport-server/migrations/60_known_ticket_names.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE known_ticket_types
+ADD COLUMN event_name VARCHAR NOT NULL DEFAULT '';
```

Comment:
small nit/note but I feel like the `DEFAULT ''` has some chance of biting us in the future.